dandi/bids_validator_xs.py

```diff
@@ -450,7 +450,7 @@ def validate_all(
 
 def write_report(
     validation_result,
-    report_path="/var/tmp/bids-validator-report_{}.log",
+    report_path="/var/tmp/bids-validator/report_{}.log",
```

Comment:
why not write this in the same directory where dandi-cli puts its logs?

Reply:
+1 on that -- there should be no hardcoded OS-specific paths. ATM we (still, might switch to platformdirs later) use appdirs:
```
$> git grep appdi
dandi/cli/command.py:import appdirs
dandi/cli/command.py:    logdir = appdirs.user_log_dir("dandi-cli", "dandi")
```

Reply:
fixed and added better logger support as well.